### PR TITLE
Add Tizen 5.0 RID for NETCoreApp

### PR DESCRIPTION
**Pull request by gbalykov**
Related PR: #29684

cc @alpencolt @Dmitri-Botcharnikov

### CODE REVIEW

File: pkg/Microsoft.Private.CoreFx.NETCoreApp/netcoreapp.rids.props [hunk 1]
```diff
@@ -28,5 +28,8 @@
     <OfficialBuildRID Include="tizen.4.0.0-armel">
       <Platform>armel</Platform>
     </OfficialBuildRID>
+    <OfficialBuildRID Include="tizen.5.0.0-armel">
```
Comment (weshaggard) [inline pkg/Microsoft.Private.CoreFx.NETCoreApp/netcoreapp.rids.props]: Should we just replace the tizen.4.0.0-armel with tizen.5.0.0-armel?

Comment (gbalykov) [reply pkg/Microsoft.Private.CoreFx.NETCoreApp/netcoreapp.rids.props]: @weshaggard both RIDs are required

Comment (weshaggard) [reply pkg/Microsoft.Private.CoreFx.NETCoreApp/netcoreapp.rids.props]: Sounds good. That implies you still support both 4.0.0 and 5.0.0 with the current master builds. Are there plans to support the latest .NET Core on 5.0.0 only in the future?

Comment (gbalykov) [reply pkg/Microsoft.Private.CoreFx.NETCoreApp/netcoreapp.rids.props]: @weshaggard until the official release of Tizen 5.0 in autumn we have to support both. After that it is yet to be decided.

**Pull request merged**